src/amo/components/LandingPage.js

```diff
@@ -108,9 +111,32 @@ export class LandingPageBase extends React.Component {
     }
 
     const { addonType, html } = content;
+    const themeText = i18n.gettext('Change your browser’s appearance. Choose from thousands of themes to give Firefox the look you want.');
+    const extensionsText = i18n.gettext('Install powerful tools that make browsing faster and safer, add-ons make your browser yours.');
 
     return (
       <div className={classNames('LandingPage', `LandingPage-${addonType}`)}>
+
+        <div className="LandingPage-Header">
+          <Icon name={classNames(`${addonType}`)} />
+          <div className="LandingPage-Header-Text">
+            <h1 className="LandingPage-Heading">
+              {addonType === 'persona' ? i18n.gettext('Themes') : i18n.gettext('Extensions')}
+            </h1>
+            <p className="LandingPage-Heading-Content">
+              {addonType === 'persona' ? i18n.gettext(themeText) : i18n.gettext(extensionsText)}
+            </p>
+          </div>
+        </div>
+
+        <Link className="Browse-Button" appearance="light"
```

Comment:
This should actually use `Button` from 'ui/components/Button'. You'll need to add support for the light appearance too. Something like:

```diff
diff --git a/src/ui/components/Button/index.js b/src/ui/components/Button/index.js                                                                                                                                                            
index 6b9e0160..b02c89cc 100644                                                                                                                                                                                                               
--- a/src/ui/components/Button/index.js                                                                                                                                                                                                       
+++ b/src/ui/components/Button/index.js                                                                                                                                                                                                       
@@ -6,6 +6,7 @@ import './Button.scss';                                                                                                                                                                                                       
                                                                                                                                                                                                                                              
 export default class Button extends React.Component {                                                                                                                                                                                        
   static propTypes = {                                                                                                                                                                                                                       
+    appearance: PropTypes.string,                                                                                                                                                                                                            
     children: PropTypes.node,                                                                                                                                                                                                                
     className: PropTypes.string,                                                                                                                                                                                                             
     href: PropTypes.string,                                                                                                                                                                                                                  
@@ -13,14 +14,16 @@ export default class Button extends React.Component {                                                                                                                                                                     
   }                                                                                                                                                                                                                                          
                                                                                                                                                                                                                                              
   static defaultProps = {                                                                                                                                                                                                                    
+    appearance: undefined,                                                                                                                                                                                                                   
     size: 'normal',                                                                                                                                                                                                                          
   };                                                                                                                                                                                                                                         
                                                                                                                                                                                                                                              
   render() {                                                                                                                                                                                                                                 
-    const { children, className, href, size, ...rest } = this.props;                                                                                                                                                                         
+    const { appearance, children, className, href, size, ...rest } = this.props;                                                                                                                                                             
     const props = {                                                                                                                                                                                                                          
       className: classNames('Button', className, {                                                                                                                                                                                           
         'Button--small': size === 'small',                                                                                                                                                                                                   
+        'Button--light': appearance === 'light',                                                                                                                                                                                             
       }),                                                                                                                                                                                                                                    
       ...rest,                                                                                                                                                                                                                               
     };